doc/source/reference/c-api/iterator.rst

```diff
@@ -370,7 +370,7 @@ Construction and Destruction
             arrays or structured arrays containing an object type)
             may be accepted and used in the iterator.  If this flag
             is enabled, the caller must be sure to check whether
-            :c:func:`NpyIter_IterationNeedsAPI(iter)` is true, in which case
```

Comment:
Could maybe keep the argument by using `:c:expr:` here

Reply:
Using `:c:expr:`, this raises a warning: `WARNING: c:identifier reference target not found: iter`. I think this reference should be removed (``` ``NpyIter_IterationNeedsAPI(iter)`` ```).